pkg/common/common.go

```diff
@@ -115,3 +123,11 @@ var (
 	// Default timeout of 4 hours to wait before resuming the testrun
 	DefaultPauseTimeout = 14400
 )
+
+func MarshalMap(annotations map[string]string) string {
```

Comment:
should be better in utils as common currently only contains definitions.

Doc is missing

Reply:
right